### ISSUE TITLE
Powershell error which leads to build failure

### ISSUE THREAD

**Issue by Chan9390** [rapid7/metasploitable3#241]
## Issue Description
When I used the latest version of Packer to build the system, I go the following error after the first restart:

```
==> virtualbox-iso: Machine successfully restarted, moving on
==> virtualbox-iso: Pausing 1m0s before the next provisioner...
==> virtualbox-iso: Provisioning with Powershell...
==> virtualbox-iso: Provisioning with powershell script: /tmp/packer-powershell-provisioner156934907
==> virtualbox-iso: Deregistering and deleting VM...
==> virtualbox-iso: Deleting output directory...
Build 'virtualbox-iso' errored: Error processing command: Error uploading ps script containing env vars: scp: ./${env:SYSTEMROOT}\Temp\packer-env-vars-5aa7cb8e-9d29-a417-5367-fb2554dca24d.ps1: No such file or directory
```

## Host System
* Packer Version: v1.2.0
* Vagrant Version: 2.0.2
* VirtualBox Version: 5.2.6

**Comment by Chan9390** [comment]
This issue is being tracked (https://github.com/hashicorp/packer/issues/5963) and will be resolved in their next build.

**Comment by Chan9390** [comment]
Same goes with Packer v1.2.1

**Comment by DanHam** [comment]
@Chan9390 There are some binaries that have been made available early by the Packer devs with the fix applied - they are available [HERE](https://github.com/hashicorp/packer/issues/5963#issuecomment-371884330) for Linux and OSX and [HERE](https://github.com/hashicorp/packer/issues/5963#issuecomment-371918969) for Windows.

As you point out the fix should be in the next release - Packer v1.2.2. 

Incidently, the root cause of this is with the SSH communicator rather than the Powershell provisioner. This is a deeper issue that needs investigating further; The fix works around this issue for the time being.

Hope that helps

**Comment by Chan9390** [comment]
Thanks @DanHam , I came across that. They are dev versions of Packer, and not yet stable. I raised this issue so that other users facing the same problem could understand the cause.

**Comment by Chan9390** [comment]
Closing the error as the bug is fixed in Packer v1.2.2
